lib/jdomodels/src/test/java/org/sagebionetworks/repo/model/dbo/dao/migration/MigratableTableDAOImplAutowireTest.java

```diff
@@ -309,4 +323,175 @@ public void testGetPrimaryMigrationTypes(){
 		System.out.println(primary);
 		assertEquals(expectedPrimaryTypes, primary);
 	}
+	
+	@Test
+	public void testGetMigrationTypeCount() throws Exception {
+		long startCount = fileHandleDao.getCount();
+		assertEquals(startCount, migratableTableDAO.getCount(MigrationType.FILE_HANDLE));
+		S3FileHandle handle = TestUtils.createS3FileHandle(creatorUserGroupId);
+		handle.setFileName("handle");
+		handle = fileHandleDao.createFile(handle);
+		filesToDelete.add(handle.getId());
+		assertEquals(startCount+1, migratableTableDAO.getCount(MigrationType.FILE_HANDLE));
+		fileHandleDao.delete(handle.getId());
+		assertEquals(startCount, migratableTableDAO.getCount(MigrationType.FILE_HANDLE));
+	}
+	
+	@Test

```

Comment:
Test should only validate that the parameters are mapped OK.
Split one case per test.